public/index.html

```diff
@@ -12,8 +20,9 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Outfit:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="manifest" href="<%= BASE_URL %>manifest.json">
   </head>
-  <body>
+  <body style="background-color: #0E0E0F">
```

Comment:
This makes the app load the first time with a dark background, a bit nicer to the eyes imo :relieved: